Shared/Samples/Apply unique value renderer/ApplyUniqueValueRendererView.swift

```diff
@@ -0,0 +1,99 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ApplyUniqueValueRendererView: View {
+    /// A Boolean value indicating whether to show an alert.
+    @State private var isShowingAlert = false
+    
+    /// The error shown in the alert.
+    @State private var error: Error? {
+        didSet { isShowingAlert = error != nil }
+    }
+    
+    /// A map with topographic basemap centered on western United States.
+    @State private var map: Map = {
```

Comment:
Playing 😈's advocate here. Instead of adding the feature layer in the task modifier, it can be added during the map's creation.

This works because the unique value renderer renders locally, without communicating with the backend.

This way everything in the task block as well as the error can be cleared away.

```swift
    /// A map with topographic basemap centered on western United States.
    @State private var map: Map = {
        let map = Map(basemapStyle: .arcGISTopographic)
        let centerPoint = Point(x: -12356253.6, y: 3842795.4, spatialReference: .webMercator)
        map.initialViewpoint = Viewpoint(center: centerPoint, scale: 52681563.2)
        
        // Create a service feature table from the census feature service.
        let featureTable = ServiceFeatureTable(url: URL(string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Census/MapServer/3")!)
        
        // Create a feature layer from the service feature table.
        let featureLayer = FeatureLayer(featureTable: featureTable)
        
        // Set a unique value renderer to the feature layer.
        featureLayer.renderer = makeUniqueValueRenderer()
        
        // Add the feature layer to the map.
        map.addOperationalLayer(featureLayer)
        
        return map
    }()
```